bin/run-couchdb.sh

```diff
@@ -15,5 +15,6 @@ echo -n "starting: $NAME "
 docker run $FLAGS \
 	--net $NETWORK \
 	-h $NAME \
+	-v "$(pwd)/couchdb_data:/usr/local/var/lib/couchdb:rw" \
```

Comment:
Well, I'd suggest to use docker volume for that. E.g. create another container couchdb-volume, and mount it. I don't like the idea of data residing on host beyond docker control.